### PR TITLE
Add job to send events to webhook

### PR DESCRIPTION
**Pull request by acasajus**

### CODE REVIEW

File: app/jobs/send_event_job.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    def __init__(self, user: User, event: EventContent):
+        self._user: User = user
+        self._event: EventContent = event
+
+    def run(self, sink: EventSink) -> bool:
+        # Check if the current user has a partner_id
+        try:
+            proton_partner_id = get_proton_partner().id
+        except ProtonPartnerNotSetUp:
+            return None
```
Comment (cquintana92) [inline app/jobs/send_event_job.py:31]:
```suggestion
            return False
```

File: app/jobs/send_event_job.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        # It has. Retrieve the information for the PartnerUser
+        partner_user = PartnerUser.get_by(
+            user_id=self._user.id, partner_id=proton_partner_id
+        )
+        event = event_pb2.Event(
```
Comment (cquintana92) [inline app/jobs/send_event_job.py:37]: Missing check of `partner_user` being `None`

File: app/jobs/send_event_job.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        user = User.get(job.payload["user_id"])
+        if not user:
+            return None
+        event_data = base64.b64decode(job.payload["event"])
+        event = event_pb2.EventContent()
+        event.ParseFromString(event_data)
+
+        return SendEventToWebhookJob(user=user, event=event)
+
+    def store_job_in_db(self, run_at: Optional[arrow.Arrow]) -> Optional[Job]:
```
Comment (cquintana92) [inline app/jobs/send_event_job.py:58]: How can the function return `None`?

Comment (acasajus) [reply app/jobs/send_event_job.py]: fixed

File: app/jobs/send_event_job.py (hunk 1 above)
Comment (acasajus) [reply app/jobs/send_event_job.py]: Fixed

**Review by cquintana92 — APPROVED** [review]

**Pull request merged**